README.md

```diff
@@ -192,7 +195,7 @@ Closing database connection.
 `csv2db` can generate the SQL statement for a staging table for your data using the `generate` command:
 
 ```sql
-$ ./csv2db generate -f test/resources/201811-citibike-tripdata.csv
+$ ./csv2db generate -f test/resources/201811-citibike-tripdata.csv -o mysql
```

Comment:
I don't think this should change in the ReadMe. For the majority of use cases, running without the database type should be sufficient to generate the table DDL

Reply:
Thats fair. Sorry for changing this. Fixed in https://github.com/csv2db/csv2db/pull/51/commits/5a890c37c8575a6b6d95bc3f418fe8ef682e672f.